porter/config.go

```diff
@@ -0,0 +1,41 @@
+package porter
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/carolynvs/magex/xplat"
+)
+
+var (
+	// DefaultMixinVersion is the default version of mixins installed when it's not present
+	DefaultMixinVersion = "canary"
+)
+
+// GetPorterHome determines the current PORTER_HOME directory
+func GetPorterHome() string {
```

Comment:
Maybe this should be an available function in Porter at some point for but now this makes it accessible to everyone without having to bring in Porter (which is a big dependency).